src/apps/dashboard/routes/index.tsx

```diff
@@ -94,6 +99,7 @@ export const Component = () => {
                                 onScanLibrariesClick={onScanLibraries}
                                 onRestartClick={promptRestart}
                                 onShutdownClick={promptShutdown}
+                                isScanning={librariesTask?.State == TaskState.Running}
```

Comment:
Should we also check if a scan is still cancelling? I imagine we should keep disabled until it's idle.

```TS
librariesTask?.State == TaskState.Cancelling
```

Suppose an alternative approach could be to check if _not_ idle.

```TS
librariesTask?.State !== TaskState.Idle
```